apps/vax/test/integration/adapter_test.exs

```diff
@@ -60,6 +60,47 @@ defmodule Vax.AdapterIntegrationTest do
     assert 30 = TestRepo.read_counter(id)
   end
 
+  test "transactional behaviour" do
+    id = Ecto.UUID.generate()
+    parent = self()
+
+    p1 =
+      spawn(fn ->
+        assert_receive {:start, p2}
+
+        TestRepo.transaction(fn ->
+          assert 0 = TestRepo.read_counter(id)
+          assert :ok = TestRepo.increment_counter(id, 10)
+          assert 10 = TestRepo.read_counter(id)
+          send(p2, :continue)
+          assert_receive :finish
+        end)
+
+        send(parent, :transaction_1_finished)
+        send(p2, :finish_2)
+      end)
+
+    _p2 =
+      spawn(fn ->
+        TestRepo.transaction(fn ->
+          send(p1, {:start, self()})
+          assert_receive :continue
+          assert 0 = TestRepo.read_counter(id)
+          assert :ok = TestRepo.increment_counter(id, 20)
+          assert 20 = TestRepo.read_counter(id)
+          send(p1, :finish)
+          assert_receive :finish_2
+        end)
+
+        send(parent, :transaction_2_finished)
+      end)
+
+    assert_receive :transaction_1_finished, 5000
+    assert_receive :transaction_2_finished, 5000
+
+    assert 30 = TestRepo.read_counter(id)
```

Comment:
When calling `antidotec_pb:commit_transaction/2`, for transaction 2, I get `{error, aborted}`. Transaction 1 commits as expected.

Reply:
ClockSI requires that all transactions on the same key(s) at one DC are serialized. This is why the abort is triggered.

Reply:
In the Antidote test suite, all txns are therefore restarted until they successfully commit.

Reply:
>ClockSI requires that all transactions on the same key(s) at one DC are serialized. This is why the abort is triggered.
Guess that alternatively we can disable this behaviour? Did that in https://github.com/vaxine-io/vaxine/pull/27/commits/93591ab8d89b5fceb35118b3e468b8457db55354